.circleci/config.yml

```diff
@@ -260,6 +260,11 @@ jobs:
     steps:
       - attach_workspace:
           at: .
+      - run:
+          name: Audit checksums of color files
+          command: |
+            chmod 744 ./checkcolorhash.sh
```

Comment:
More of a wondering, I know the other scripts change permissions before running, but from a quick google it seems like git should track if a file is executable. Is there a way we can remove the `chmod` line and just run the script? Wouldn't spend much time, more of a curiosity then anything to spend a lot of time on.

Reply:
![Screen Shot 2022-05-10 at 8 42 38 AM](https://user-images.githubusercontent.com/3288586/167630917-b4d04161-be0c-436b-85f4-2a0645c6963d.png)

Tried it without chmod and got this. I wonder if its more of a Circle CI thing than a git thing. Sarah Jaine would probably be able to tell us the full story.